app/helpers/email_helper.rb

```diff
@@ -1,5 +1,9 @@
 module EmailHelper
   def current_subdomain
-    Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current 
+    if Hours.single_tenant_mode?
+      ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.